packages/eas-cli/src/credentials/ios/appstore/bundleIdCapabilities.ts

```diff
@@ -733,6 +733,13 @@ export const CapabilityMapping: CapabilityClassifier[] = [
     validateOptions: validateBooleanOptions,
     getOptions: getBooleanOptions,
   },
+  {
+    entitlement: 'com.apple.developer.proximity-reader.payment.acceptance',
+    name: 'Tap to Pay on iPhone',
+    capability: 'TAP_TO_PAY_ON_IPHONE' as CapabilityType,
```

Comment:
I've published `@expo/apple-utils@1.3.2` with `TAP_TO_PAY_ON_IPHONE`, `SUSTAINED_EXECUTION`, and `NETWORK_SLICING`.

Reply:
Thank you! The PR is updated to reflect this change.